eng/Build.props

```diff
@@ -39,6 +39,10 @@
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Json.6.0.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Json.8.0.4.csproj" />
 
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Encodings.Web.6.0.1.csproj" />
```

Comment:
Friendly reminder to add these DependencyPackageProjects whenever adding new packages.  This is documented [here](https://github.com/dotnet/source-build-reference-packages/blob/main/README.md#workflow).